command/operator_debug.go

```diff
@@ -1307,6 +1446,53 @@ func stringToSlice(input string) []string {
 	return out
 }
 
+func parseEventTopics(topicList []string) (map[api.Topic][]string, error) {
+	topics := make(map[api.Topic][]string)
+
+	for _, topic := range topicList {
+		k, v, err := parseTopic(topic)
+		if err != nil {
+			return nil, fmt.Errorf("error parsing topics: %w", err)
+		}
+
+		topics[api.Topic(k)] = append(topics[api.Topic(k)], v)
```

Comment:
Should we use `strings.Title()` to ensure the topic key starts with a capital letter? The events stream is case sensitive, resulting in allocation not working and Allocation working. This tripped me up and I can imagine it would do the same to others.